validator/validators/stress/stress_validator.go

```diff
@@ -113,11 +113,11 @@ var (
 				"net_packets_sent":     float64(700),
 			},
 			"logs": {
-				"procstat_cpu_usage":   float64(225),
```

Comment:
Sending logs during the [first minute changing the logs stress metrics ](https://github.com/aws/amazon-cloudwatch-agent-test/pull/165/files#diff-32ea2f52d0d817b42a874abb6e550c6c2040c34707954b8efd66e0c474660738R57)